vigoda/main.go

```diff
@@ -26,7 +26,7 @@ func main() {
 			fmt.Fprintf(w, "error executing template: %v\n", err)
 			return
 		}
-
+		log.Printf("Yep, still dead.")
```

Comment:
👀

Reply:
I wanted some logs so I could verify things were working, do you object to me leaving it in?

Reply:
Oh, no I didn't realize that. :)